guides/v2.3/extension-dev-guide/build/create_component.md

```diff
@@ -27,7 +27,7 @@ The smallest working `module.xml` file would look something like this:
 </config>
 ```
 
-The `name` parameter defines the name of your component. It is required for all components. If you do not use [Declarative Schema]({{ page.baseurl }}/extension-dev-guide/declarative-schema/index.html)  to help manage the installation and upgrade processes for your component, then you must also add the  `setup_version` parameter to the `module` line. Set the `setup_version` value to your module's [database schema](https://glossary.magento.com/database-schema) version. Omit the `setup_version` parameter if you implement Declarative Schema.
+The `name` parameter defines the name of your component. It is required for all components. If you do not use [Declarative Schema]({{ page.baseurl }}/extension-dev-guide/declarative-schema/index.html)  to help manage the installation and upgrade processes for your component, then you must also add the  `setup_version` parameter to the `module` line. Set the `setup_version` value to your module's [database schema](https://glossary.magento.com/database-schema) version. Omit the `setup_version` parameter if you no implement Declarative Schema.
```

Comment:
This contradicts what we're saying in the third sentence:

> If you do not use [Declarative Schema]({{ page.baseurl }}/extension-dev-guide/declarative-schema/index.html)  to help manage the installation and upgrade processes for your component, then you must also add the  `setup_version` parameter to the `module` line.

Reply:
It should be the opposite. I will change it . If you don't use Declarative Schema you don't need add the attribute `setup_version`. If you use it you need add the attribute `setup_version`